src/hotspot/share/gc/serial/serialFullGC.cpp

```diff
@@ -604,11 +609,34 @@ template <class T> void SerialFullGC::mark_and_push(T* p) {
   }
 }
 
+inline size_t SerialFullGC::adjust_pointers(oop obj) {
```

Comment:
I see only a single caller of this method; maybe this method can be inlined completely and drop this api.